src/OrderLinesList/Details/OrderLineDetails.js

```diff
@@ -1,118 +1,163 @@
-import React, { Component, Fragment } from 'react';
+import React, { Fragment, useCallback, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import ReactRouterPropTypes from 'react-router-prop-types';
 import { get } from 'lodash';
 
 import {
   baseManifest,
+  LoadingPane,
   Tags,
 } from '@folio/stripes-acq-components';
 
-import { POLineView } from '../../components/POLine';
 import {
   ORDERS_API,
   LINES_API,
 } from '../../components/Utils/api';
-
+import { POLineView } from '../../components/POLine';
 import { FILTERS as ORDER_FILTERS } from '../../OrdersList';
 
-class OrderLineDetails extends Component {
-  static manifest = Object.freeze({
-    order: {
-      ...baseManifest,
-      path: (queryParams, pathComponents, resourceData, logger, props) => {
-        const orderId = get(props, ['resources', 'orderLine', 'records', 0, 'purchaseOrderId']);
-
-        return orderId ? `${ORDERS_API}/${orderId}` : null;
-      },
+const OrderLineDetails = ({
+  match,
+  mutator,
+  onClose,
+  parentMutator,
+  parentResources,
+  resources,
+  showToast,
+  ...restProps
+}) => {
+  const lineId = match.params.id;
+  const line = get(resources, ['orderLine', 'records', 0], {});
+  const order = get(resources, ['order', 'records', 0], {});
+
+  const fetchLineDetails = useCallback(
+    () => {
+      mutator.orderLine.reset();
+      mutator.orderLine.GET();
+      mutator.order.reset();
+      mutator.orderLine.GET()
```

Comment:
the same as above (might be wrong mutator)

Reply:
fixed